logging/setup.py

```diff
@@ -52,12 +52,12 @@
 REQUIREMENTS = [
     'google-cloud-core >= 0.23.1, < 0.24dev',
     'grpcio >= 1.0.2, < 2.0dev',
-    'gapic-google-cloud-logging-v2 >= 0.90.1, < 0.91dev',
+    'gapic-google-cloud-logging-v2 >= 0.91.0, < 0.92dev',
 ]
 
 setup(
     name='google-cloud-logging',
-    version='0.23.0',
```

Comment:
Don't modify this unless you are creating a release from this PR. By bumping versions in non-release PRs I had a **lot more** to check when doing last week's release (the worst case, Spanner had been bumped twice from `0.23.0` to `0.23.1` to `0.23.2` without any release in the middle)